src/components/RepositoriesList/RepositoriesList.tsx

```diff
@@ -21,14 +23,18 @@ function RepositoriesList() {
   }, [selectedRepositiry]);
 
   useEffect(() => {
+    setIsLoading(true);
```

Comment:
why do you need to set this state to true before fetching if by default it is true?